documentation/source/user_section/tutorials/processing-fmri-data/motion-correction-for-fmri.rst

```diff
@@ -39,3 +39,23 @@ To apply the algorithm, we use the :ref:`sct_fmri_moco` command:
 
 .. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/processing-fmri-data/io-sct_fmri_moco.png
    :align: center
+
+
+Checking QC for Temporal Signal-to-Noise Ratio (tSNR)
+-----------------------------------------------------
+
+Here, we can generate a `tSNR` map for the original and motion-corrected fMRI data. This map is a measure of the signal-to-noise ratio in the temporal domain. The higher the tSNR, the better the quality of the data.
```

Comment:
since during the course, we noticed that the interpolation has an impact on this, do we want to mention it?

Reply:
From meeting: yes, it would be good to add the tSNR QC report comparison in the tutorials. But, it can be a separate PR.